assets/js/unminified/frontend.js

```diff
@@ -613,7 +613,8 @@ var astraTriggerEvent = function astraTriggerEvent( el, typeArg ) {
 	AstraToggleSetup = function () {
 
 		if ( 'off-canvas' === mobileHeaderType || 'full-width' === mobileHeaderType ) {
-			var __main_header_all = document.querySelectorAll( '#ast-mobile-popup' );
+			// comma seprated selector added, if menu is outside of off-canvas then submenu will not clickable, it work only for off-cnavas area submenu.
```

Comment:
```suggestion
			// comma separated selector added, if menu is outside of Off-Canvas then submenu is not clickable, it work only for Off-Canvas area with dropdown style.
```

Please add changelog as well.

Reply:
Comment updated also added changelog for the same just confirm the version that I have added.